packages/react-charting/docs/TestPlans/AreaChart/ComponentTests.md

```diff
@@ -0,0 +1,60 @@
+|                   - HideLegend prop set to “false”                    |                                  Should mount legend when hideLegend is false                                   |    Enzyme     |
+|                   - HideTooltip prop set to “true”                    |                                Should not mount callout when hideTootip is true                                 |    Enzyme     |
+|                   - HideTooltip prop set to “false”                   |                                  Should mount callout when hideTootip is false                                  |    Enzyme     |
+|              - onRenderCalloutPerStack prop is not given              |                                    Should not render onRenderCalloutPerStack                                    |    Enzyme     |
+|                - onRenderCalloutPerDataPoint is given                 |                                    Should render onRenderCalloutPerDataPoint                                    |    Enzyme     |
+|              - onRenderCalloutPerDataPoint is not given               |                                  Should not render onRenderCalloutPerDataPoint                                  |    Enzyme     |
+|            Test 3: Render calling with respective to props            |                                                                                                                 |               |
+| - No prop changes: Mount Area chart and then set the same props again |                                  Render function should have been called twice                                  |    Enzyme     |
+|     - Prop changes: Mount Area chart and then set some other prop     |                                  Render function should have been called twice                                  |    Enzyme     |
+|                         Test 4: Mouse events                          |                                                                                                                 |               |
+|                         - Mouse over on Area                          |                                  Should render callout correctly on mouseover                                   |    Enzyme     |
+|               - Mouse move from one area to other area                |                                  Should render callout correctly on mouse move                                  |    Enzyme     |
+|            - Mouse over on a area with customized callout             |                                  Should render customized callout on mouseover                                  |    Enzyme     |
+|                     - Customized callout on area                      |                             Should render customized callout correctly on mouseover                             |    Enzyme     |
```

Comment:
what is the difference between Mouse over on a area with customized callout and Customized callout on area